.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:9.5.0
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run:
+          name: Install global packages
+          command: sudo npm install -g truffle codecov
```

Comment:
I haven't used codecov before.  Is this part of circleci integration? I created a codecov account not sure if I should add c4coin organization.

Reply:
CodeCov is excellent.  It's also free for public repositories but won't work until we have code to actually test.